src/main/java/apprenticeship/App.java

```diff
@@ -1,13 +1,24 @@
 package apprenticeship;
 
+import java.util.Arrays;
+
 /**
  * Hello world!
  *
  */
 public class App 
 {
+
     public static void main( String[] args )
     {
-        System.out.println( "Hello World!" );
+
+        char[] array = new char[9];
```

Comment:
The number 9 is a "magic number" - a reader of your code might not know where it comes from. I would add a constant at the top of the file. Something like `NumberOfCells = 9` (however you do that in Java) and then write `char[] array = new char[NumberOfCells]`